partner_event/views/res_partner_view.xml

```diff
@@ -20,6 +20,7 @@
         <div name="button_box" position="inside">
             <button name="%(partner_event.act_partner_registration)d"
                     type="action"
+                    groups="event.group_event_user"
```

Comment:
I said in the view, not in the element. This still arises the error.

Reply:
done